### DISCUSSION TITLE
An error occurred @prisma/client did not initialize yet. Please run \"prisma generate\" and try to import it again.

### DISCUSSION THREAD

**Discussion by aminsoraya** [prisma/prisma#21342] [Q&A]
### Bug description

I have rebind prisma connection string to my own server and I regenerate new schema . Everything is okay in development mode. But when I deploy a project . Bellow Error shown in console :

**"An error occurred @prisma/client did not initialize yet. Please run \"prisma generate\" and try to import it again.\nIn case this error is unexpected for you, please report it in https://github.com/prisma/prisma/issues"**.

What is it and how can I fixt it .

**Edit**

Here are snipped which I used prisma in main component :

```
export const getServerSideProps: GetServerSideProps = async () => {
 
    // Replace this with your actual Prisma query
    const response = await Prisma().$queryRaw`call GetArayeshgahByCategories()`;
    
    return {
      props: {
        data: JSON.stringify(dataWithBlurImages),
      },
    };
  } 
```

Prisma Component :

```
import React from "react";
import { PrismaClient } from "@prisma/client";

export default function Prisma() {
  // let prisma = new PrismaClient({
  //   log: [
  //     {
  //       emit: "stdout",
  //       level: "query",
  //     },
  //     {
  //       emit: "stdout",
  //       level: "error",
  //     },
  //     {
  //       emit: "stdout",
  //       level: "info",
  //     },
  //     {
  //       emit: "stdout",
  //       level: "warn",
  //     },
  //   ],
  // });

  let prisma = new PrismaClient();

  return prisma;
}
```

### Expected behavior

return result as expected

### Prisma information

```prisma
// Add your schema.prisma
```

```ts
// Add your code using Prisma Client
```

### Environment & setup

- OS: Win 10
- Database: MySQL
- Node.js version: v20.1.0

### Prisma Version

```
"prisma": "^5.3.1",
```

**Comment by nurul3101** [comment]
Hello @aminsoraya 👋 

Can you share how your deployment script looks like?

You would need to make sure that `npx prisma generate` command is a part of your deployment script.

Here's a reference example: https://github.com/prisma/deployment-example-vercel/blob/main/package.json#L13

Can you try it out?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
